_includes/card.html

```diff
@@ -10,14 +10,15 @@
 
 
 <li class="usa-card {{ wds_grid_class }}">
-    <div class="usa-card__container" data-href="/site/what-we-deliver/forest-service/" tabindex="-1">
+    <div class="usa-card__container shadow-2" data-href="/site/what-we-deliver/forest-service/" tabindex="-1">
       
       <header class="usa-card__header">
         {% if include.agency %}
-          <h6 class="card-tag">{{ include.agency }}</h6>
+          <h6 class="font-sans-8 text-bold text-base-darker
```

Comment:
in cases where you're adding more than 2 uswds utility classes, I suggest making those changes in the css using the uswds mixins.